### PR TITLE
Fix name and autosave

### PR DESCRIPTION
**Pull request by klauer**
**This is as-deployed**

*  Moved to ioc-kfe-vac and synchronized IOC manager
*  Bumped to ads-ioc R0.5.0
* Fixed autosave by giving the IOC user `ps-ioc` access to `$(IOC_TOP)/autosave`

### CODE REVIEW

File: plc/iocBoot/ioc-kfe-vac/envPaths [hunk 1]
```diff
@@ -1,14 +1,14 @@
-epicsEnvSet("IOC","ioc-plc-kfe-vac")
-epicsEnvSet("TOP","/reg/g/pcds/epics/ioc/common/ads-ioc/R0.4.1")
+epicsEnvSet("IOC","ioc-kfe-vac")
```
Comment (ghalym) [inline plc/iocBoot/ioc-kfe-vac/envPaths:1]: I thought the IOC should have the PLC project name in it?

**Comment by ZLLentz** [comment]
> Moved to ioc-lfe-vac

Is this a typo in the PR description?

**Comment by ZLLentz** [comment]
Also, does the `chown` operation get committed in git? I didn't realize this if so, one of your commits implies that this is the case

**Comment by klauer** [comment]
> Is this a typo in the PR description?

I had fixed these both at the same time and the PR text is the only copy/paste issue (hopefully)

> I thought the IOC should have the PLC project name in it?

I think it should, but I recall trying to make this match the others:
https://confluence.slac.stanford.edu/display/PCDS/EPICS+IOCs+Deployed+in+IOC+Manager
More didn't follow that convention than did:
image

File: plc/iocBoot/ioc-kfe-vac/envPaths (hunk 1 above)
Comment (ZLLentz) [reply plc/iocBoot/ioc-kfe-vac/envPaths:1]: It looks like the iocmanager calls this ioc `ioc-kfe-vac` though, is that what's going on here? Some issues being caused by the iocmanager name and the name here being different?

Comment (klauer) [reply plc/iocBoot/ioc-kfe-vac/envPaths:1]: Yes, they need to be consistent. And we had to choose one name

**Comment by ZLLentz** [comment]
> Fixed autosave by giving the IOC user ps-ioc access to $(IOC_TOP)/autosave

According to my googling, as far as file permissions go only the execute bit gets committed by git. So I don't think this last bullet point here can be solved in the way you've described.

**Comment by klauer** [comment]
> So I don't think this last bullet point here can be solved in the way you've described.

I guess I wasn't really indicating it was solved in the repo, just what was done locally to fix the IOC

**Comment by ZLLentz** [comment]
Ok, so this is a marked improvement then: for the current (dev) install, autosave works properly.
We'd need to do some more work in a follow-up for a tagged install to autosave properly, right?
Possibly related to https://github.com/pcdshub/ads-ioc/issues/89 ?

**Review by ZLLentz — APPROVED** [review]

**Comment by ZLLentz** [comment]
There's some funky stuff going on in the CI for the docs build- not sure how much weight to put on that failure

**Comment by klauer** [comment]
> Ok, so this is strictly an improvement then: for the current (dev) install, autosave works properly.

Yes, exactly. 

I hope that the new ads-ioc R0.6.0 should make this a problem of the past...

**Comment by klauer** [comment]
> There's some funky stuff going on in the CI for the docs build- not sure how much weight to put on that failure

Another pytmc bug, great!

**Comment by klauer** [comment]
Docs building bug resolved and CI is now passing; merging

**Pull request merged**